src/main/java/ua/danit/final_project/controllers/WorkShiftController.java

```diff
@@ -53,6 +53,7 @@ public WorkShiftController(WorkCommentService workCommentService,
   @PostMapping("/comment")
   public ShiftCommentDto createComment(@RequestBody ShiftCommentDto shiftCommentDto) throws JsonProcessingException {
     User userFromToken = getCurrentUser();
+    ShiftCommentDto ex = shiftCommentDto;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "ex". ([squid:S1854](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=167&issues=AWbwEn5XsSodgw_KsuuF&open=AWbwEn5XsSodgw_KsuuF)

[ISSUE_KEY]: AWbwEn5XsSodgw_KsuuF